request-server_test.go

```diff
@@ -39,6 +40,9 @@ func clientRequestServerPair(t *testing.T) *csPair {
 	skipIfPlan9(t)
 	ready := make(chan bool)
 	os.Remove(sock) // either this or signal handling
+	pair := &csPair{
+		svrResult: make(chan error),
```

Comment:
Can you make this with a buffer of 1? That way even if we do not read from this channel, there is not a blocked goroutine left hanging?

Reply:
Done, thank you